github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -919,6 +912,9 @@ presubmits:
         resources:
           requests:
             memory: "29Gi"
+            prow/sriov: "2"
+          limits:
+            prow/sriov: "2"
```

Comment:
Do we still need ``` k8s.v1.cni.cncf.io/networks: 'multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni'``` when using these?

Reply:
yes, 
the annotation say to run the cni (twice)
and those hint the scheduler how many it consumes